packages/patternfly-4/react-table/src/components/Table/SortColumn.js

```diff
@@ -1,48 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { LongArrowAltUpIcon, LongArrowAltDownIcon, ArrowsAltVIcon } from '@patternfly/react-icons';
-import { css } from '@patternfly/react-styles';
-import styles from '@patternfly/react-styles/css/components/Table/table';
-
-const propTypes = {
-  children: PropTypes.node,
-  className: PropTypes.string,
-  isSortedBy: PropTypes.bool,
-  onSort: PropTypes.func,
-  sortDirection: PropTypes.string
-};
-const defaultProps = {
-  children: null,
-  className: '',
-  isSortedBy: false,
-  onSort: null,
-  sortDirection: ''
-};
-
-export const SortByDirection = {
```

Comment:
just moving the exports here. Tried to gather all of the base types in `base/types.tsx` and all of our extended types in `Table.tsx` so that they are all together for now...

Reply:
lots of types!